### PR TITLE
Production deployment Key&Verification server settings

### PR DESCRIPTION
**Pull request by jendakol**
KS: MAX_KEYS_ON_PUBLISH, MAX_SAME_START_INTERVAL_KEYS
VS: RATE_LIMIT_TOKENS

### CODE REVIEW

**Comment by marekaf** [comment]
atlantis plan

**Comment by marekaf** [comment]
@jendakol could you pls rebase this? or should I?:)

File: terraform/prod/key-server/main.tf [hunk 1]
```diff
@@ -42,6 +42,8 @@ module "en" {
     }
     exposure = {
       OBSERVABILITY_EXPORTER = "NOOP"
+      MAX_KEYS_ON_PUBLISH          = 50
+      MAX_SAME_START_INTERVAL_KEYS = 15
```
Comment (marekaf) [inline terraform/prod/key-server/main.tf:46]: this is not formatted correctly

if you have terraform, please run `terraform fmt -recursive`

**Comment by jendakol** [comment]
Formatted, rebased

**Review by marekaf — APPROVED** [review]
@jendakol looks good now. 
Atlantis is failing only because terraform can't find the efgs functions in prod account (they don't exist there as there's the dirft between master and prod tag...)

Feel free to merge

**Pull request merged**